rider/src/main/kotlin/com/jetbrains/rider/plugins/unity/notifications/OutOfSyncEditorNotification.kt

```diff
@@ -0,0 +1,27 @@
+package com.jetbrains.rider.plugins.unity.notifications
+
+import com.intellij.notification.Notification
+import com.intellij.notification.NotificationGroup
+import com.intellij.notification.NotificationType
+import com.intellij.notification.Notifications
+import com.intellij.openapi.project.Project
+import com.jetbrains.rdclient.util.idea.LifetimedProjectComponent
+import com.jetbrains.rider.plugins.unity.UnityHost
+import com.jetbrains.rider.plugins.unity.actions.InstallEditorPluginAction
+import com.jetbrains.rider.util.reactive.adviseNotNull
+
+class OutOfSyncEditorNotification(project: Project, unityHost: UnityHost): LifetimedProjectComponent(project) {
+    companion object {
+        private val notificationGroupId = NotificationGroup.balloonGroup("Unity connection is out of sync")
+    }
+
+    init {
+        unityHost.model.onEditorModelOutOfSync.adviseNotNull(componentLifetime) {
+            val message = "UnityEditor connection is out of sync, but automatic plugin update is disabled."
```

Comment:
How about?

> **Unity editor plugin update required**
>
> The Unity editor plugin is out of date and automatic plugin updates are disabled. Advanced Unity integration features are unavailable until the plugin is updated.

Reply:
Even though it doesn't reveal all the technical details, but it makes more sense for the user. Thanks!